cypress.env.json

```diff
@@ -0,0 +1,7 @@
+{
+  "HOME_URL": "https://qa-react-conduit.herokuapp.com/",
+  "USER_EMAIL": "alan.reskin@wolox.com.ar",
+  "USER_NAME": "churry",
+  "USER_PASSWORD": "wolox1189",
```

Comment:
NOOOOO I SAID DO NOT PUSH CREDENTIALS

Reply:
WHAT THE FUCK IS HAPPENING WITH GITTTT, hate it. 🤣